### ISSUE TITLE
Obtainium detects a non-existent version of Dolphin Emulator.

### ISSUE THREAD

**Issue by RuneCE** [ImranR98/Obtainium#2023]
**Prerequisites**
*Complete*

**Describe the bug**
I added https://dolphin-emu.org/download/ to Obtainium, it says I have version 2412 (latest) and then says the latest is "622797097"... I have no clue where it gets this number from, I checked the page, I checked Inspect Element, there is no "622797097" anywhere - when I do click update though, it downloads  to version 2412 and then requests I update again.

**To Reproduce**
Add App > https://dolphin-emu.org/download/ > Override Source as HTML > Add

**Screenshots and Logs**
![Screenshot_20241209_221144_Obtainium](https://github.com/user-attachments/assets/12c772c3-5216-4563-96fa-2742a3b9ae3e)

**Please complete the following information:**
 - Device: Fold 4
 - OS: Stock
 - Obtainium Version: v1.1.34 (latest)

**Additional context**
N/A

**Comment by DarkCrypt** [comment]
Your issue is because of the version mismatch as seen in your screenshot. Here's how to fix it.

Version String Extraction ReGex: `[0-9]+` or `\d+` either one produces the same results.

**Comment by DwainZwerg** [comment]
622797097 is the pseudo version created by Obtainium if real versioning is not possible.

**Comment by ImranR98** [comment]
Yes, this is a pseudo-version (read the Wiki for details). Because the pseudo-version format and actual version formats are the same (plain numbers with no sub-versioning), Obtainium makes a mistake and tries to compare the two.

This is a bug - the comparison should automatically be disabled.

For now, you should disable the "Reconcile OS version..." setting manually.

**Comment by ImranR98** [comment]
Fixed: https://github.com/ImranR98/Obtainium/releases/tag/v1.1.35

**Comment by RuneCE** [comment]
> Fixed: https://github.com/ImranR98/Obtainium/releases/tag/v1.1.35

I am on 1.1.36 and this is still a persistent issue...
And it happens wether I disable or enable "Reconcile OS version".

**Comment by DwainZwerg** [comment]
@RuneCE Could you upload a new screenshot how your screen looks now?

**Comment by DarkCrypt** [comment]
I still agree with my previous comment. All you need to do is extract the version so it'll match the installed version.

There's 2 different URLs which can be used.
https://dolphin-emu.org/download/ is the main download page with sections for stable and dev builds.
https://dolphin-emu.org/download/list/releases/1/ has only stable builds.

Config for 1st link:
Import App -> [Dolphin Emulator](https://apps.obtainium.imranr.dev/redirect?r=obtainium://app/%7B%22id%22%3A%22org.dolphinemu.dolphinemu%22%2C%22url%22%3A%22https%3A%2F%2Fdolphin-emu.org%2Fdownload%2F%22%2C%22author%22%3A%22dolphin-emu.org%22%2C%22name%22%3A%22Dolphin%20Emulator%22%2C%22preferredApkIndex%22%3A0%2C%22additionalSettings%22%3A%22%7B%5C%22intermediateLink%5C%22%3A%5B%5D%2C%5C%22customLinkFilterRegex%5C%22%3A%5C%22%5C%22%2C%5C%22filterByLinkText%5C%22%3Afalse%2C%5C%22skipSort%5C%22%3Afalse%2C%5C%22reverseSort%5C%22%3Afalse%2C%5C%22sortByLastLinkSegment%5C%22%3Afalse%2C%5C%22versionExtractWholePage%5C%22%3Afalse%2C%5C%22requestHeader%5C%22%3A%5B%7B%5C%22requestHeader%5C%22%3A%5C%22User-Agent%3A%20Mozilla%2F5.0%20(Linux%3B%20Android%2010%3B%20K)%20AppleWebKit%2F537.36%20(KHTML%2C%20like%20Gecko)%20Chrome%2F114.0.0.0%20Mobile%20Safari%2F537.36%5C%22%7D%5D%2C%5C%22defaultPseudoVersioningMethod%5C%22%3A%5C%22partialAPKHash%5C%22%2C%5C%22trackOnly%5C%22%3Afalse%2C%5C%22versionExtractionRegEx%5C%22%3A%5C%22%5B0-9%5D%2B%5C%22%2C%5C%22matchGroupToUse%5C%22%3A%5C%22%5C%22%2C%5C%22versionDetection%5C%22%3Atrue%2C%5C%22useVersionCodeAsOSVersion%5C%22%3Afalse%2C%5C%22apkFilterRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22invertAPKFilter%5C%22%3Afalse%2C%5C%22autoApkFilterByArch%5C%22%3Atrue%2C%5C%22appName%5C%22%3A%5C%22%5C%22%2C%5C%22shizukuPretendToBeGooglePlay%5C%22%3Afalse%2C%5C%22allowInsecure%5C%22%3Afalse%2C%5C%22exemptFromBackgroundUpdates%5C%22%3Afalse%2C%5C%22skipUpdateNotifications%5C%22%3Afalse%2C%5C%22about%5C%22%3A%5C%22An%20emulator%20for%20running%20GameCube%20and%20Wii%20games%20on%20Windows%2C%20Linux%2C%20macOS%2C%20and%20recent%20Android%20devices.%5C%22%2C%5C%22refreshBeforeDownload%5C%22%3Afalse%7D%22%2C%22overrideSource%22%3Anull%7D)
- App Source URL: `https://dolphin-emu.org/download/`
- Version String Extraction ReGex: `[0-9]+`
- About: `An emulator for running GameCube and Wii games on Windows, Linux, macOS, and recent Android devices.`

**Comment by RuneCE** [comment]
> @RuneCE Could you upload a new screenshot how your screen looks now?

Exactly the same as the initial image, but here you go:
![Screenshot_20241219_234826_Obtainium](https://github.com/user-attachments/assets/127b82b1-32b2-49fc-a9df-d446b22fdf8b)
![Screenshot_20241219_234834_Obtainium](https://github.com/user-attachments/assets/1c6adf3a-387d-4f0e-bfe8-218383548ece)
